human_cancer_analysis/1_process_data.ipynb

```diff
@@ -55,12 +55,44 @@
    ],
    "source": [
     "import os\n",
+    "import random\n",
+    "import tensorflow as tf\n",
     "import pandas as pd\n",
     "import pickle\n",
     "from ponyo import utils, train_vae_modules\n",
     "from generic_expression_patterns_modules import process"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set seeds to get reproducible VAE trained models\n",
```

Comment:
These lines seem to be copy/pasted in multiple places. Does it make sense to create a utility function such as `set_random_seeds()` somewhere in `generic_expression_patterns_modules/`, then import and call it in your notebooks?

Reply:
I think thats a great point. I'll make that change